lib/wrapbox/runner/ecs.rb

```diff
@@ -354,30 +356,19 @@ def create_task(task_definition_arn, class_name, method_name, args, command, par
               reason: "signal interrupted"
             )
             wait_task_stopped(cl, task.task_arn, TERM_TIMEOUT)
-            @logger.debug("Stop Task: #{task.task_arn}")
+            @logger.debug("#{log_prefix}Stop Task: #{task.task_arn}")
             nil
           end
         end
       end
 
-      def wait_until_with_timeout(cluster, task_arn, timeout, waiter_name)
-        client.wait_until(waiter_name, cluster: cluster, tasks: [task_arn]) do |w|
-          if timeout
-            w.delay = WAIT_DELAY
-            w.max_attempts = timeout / w.delay
-          else
-            w.max_attempts = nil
-          end
-        end
-      end
-
       def wait_task_running(cluster, task_arn, launch_timeout)
-        wait_until_with_timeout(cluster, task_arn, launch_timeout, :tasks_running)
+        @task_waiter.wait_task_running(task_arn, timeout: launch_timeout)
       end
 
       def wait_task_stopped(cluster, task_arn, execution_timeout)
```

Comment:
[imo] `cluster` may be called from `@task_waiter`

Reply:
I found that `task_waiter` didn't work expectedly if `cluster` was overridden.
I fixed it https://github.com/reproio/wrapbox/pull/27/commits/0a09c99bf6e42893b85b0e1c3ea8c15836c8db19

Reply:
I got it! It looks to me that your change made easier to understand it.
It would be helpful to update 'options only for ECS' in README 👀

Reply:
Although I don't know whether `run` and `run_cmd` in README means singleton methods of `Wrapbox` or instance methods of each runner, I've updated README. https://github.com/reproio/wrapbox/pull/27/commits/aa7cd28695fe7a64f46dea06f5759838fc79b9f7